ResearchKit/ActiveTasks/ORKAudioStepViewController.h

```diff
@@ -37,6 +37,8 @@ NS_ASSUME_NONNULL_BEGIN
 ORK_CLASS_AVAILABLE
 @interface ORKAudioStepViewController : ORKActiveStepViewController
 
+@property (nonatomic, assign) CGFloat alertThreshold;
```

Comment:
Updated. This was originally written by Thread in advance of promoting mPower during the March Keynote. I am not certain why they set it initially to -1, but doing so is unnecessary. Thanks for catching.